### PR TITLE
✨ Add github workflow to update nix inputs

### LINKED ISSUE #15: Add a job to automatically update nix flake
I'm not familiar with Github's workflows but it seems straightforward. Hyprland has it: https://github.com/hyprwm/Hyprland/blob/main/.github/workflows/nix-update-inputs.yml.

### PR DESCRIPTION
**Pull request by arthsmn**
Closes #15. I'm not familiar with this, so feedback is welcome.

### CODE REVIEW

File: .github/workflows/update-nix-inputs.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      	uses: actions/checkout@v3
+
+      - uses: DeterminateSystems/nix-installer-action@main
+      - name: Update inputs
+	      run: nix/update-inputs.sh
+
+      - name: Commit
+	      uses: stefanzweifel/git-auto-commit-action@v4
+	      with:
+	        commit_message: "⬆️ Nix: update inputs"
```
Comment (ewen-lbh) [inline .github/workflows/update-nix-inputs.yml:18]:
```suggestion
      - name: Commit
	uses: stefanzweifel/git-auto-commit-action@v4
	with:
	  commit_message: "⬆️ Nix: update inputs"
```

there too

File: .github/workflows/update-nix-inputs.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  update:
+    name: inputs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+      	uses: actions/checkout@v3
+
+      - uses: DeterminateSystems/nix-installer-action@main
+      - name: Update inputs
+	      run: nix/update-inputs.sh
```
Comment (ewen-lbh) [inline .github/workflows/update-nix-inputs.yml:13]: Seems like we need that script in the repo, heres Hyprland's script: https://github.com/hyprwm/Hyprland/blob/main/nix/update-inputs.sh

i know nothing about nix so I can't help on how we would adapt it though

Comment (arthsmn) [reply .github/workflows/update-nix-inputs.yml]: As it is just needed to update the inputs without checking, I've used the justfile.

**Comment by arthsmn** [comment]
@ewen-lbh do you have an idea on how to test this?

**Comment by ewen-lbh** [comment]
Unfortunately CI is notoriously annoying/impossible to test locally 🥲

Apart from running the just script locally, there's not much you can do but commit and try ^^

Let me know when you tested the just recipe locally, i'll merge this in. If it doesn't work on the first try, that's fine

**Comment by arthsmn** [comment]
> Unfortunately CI is notoriously annoying/impossible to test locally 🥲
> 
> Apart from running the just script locally, there's not much you can do but commit and try ^^
> 
> Let me know when you tested the just recipe locally, i'll merge this in. If it doesn't work on the first try, that's fine

The Justfile addition is fine.

**Pull request merged**